views/layouts/default.html.erb

```diff
@@ -29,13 +29,14 @@
             <nav class="pd-b">
               <a href="/" title="Welcome">Welcome</a>
               <a href="/#sec-hardfork" title="Bigger&nbsp;blocks">Bigger&nbsp;blocks</a>
+              <a href="/onboard.html" title="Who's Onboard">Who's&nbsp;Onboard</a>
```

Comment:
@NxtChg Well I guess the easy way is to drop some content of the menu (maybe @mikehearn would like to choose what remains there?). The hard way is to try fixing the design altogether to accommodate larger menus (seems like this would be a moderate to large change).